### ISSUE TITLE
Research on the Wins submission process

### ISSUE THREAD

**Issue by Sihemgourou** [hackforla/website#1789]
~### Dependencies
- #1551 
- #1717 
- #1845~

### Overview
We want people to share their wins on social media so that HfLA's exposure is increased and other people will join us as volunteers and/or potentially make a donation.

### Additional Detail
* Primary goal : **Create Engagement and Traffic to HFLA.org** 
As a submitter of a win, I want to be notified when my submission is published on the HFLA website (nice to have : so I can share it on my social media.)

### Action Items
- [x] Design a research plan and add a page in the wiki for it
   - [x] with the following goals:
      - [x]  how the process of placing wins can be improved 
      - [x]  how people would like to be notified when their win is accepted
   - [x] define how you will select the research participants

- [x] Review research plan with product

  - [x] Conduct Research: Wins Submission Process Volunteers
    - [x] Recruit those interested in sharing what they've learned from HfLA (both current or past volunteers are ok)

  - [x] Add links in wiki page to Analysis resources
  - [x] Present insights and recommendations to product
    - [x] Slide Deck (add to wiki)
    - [x] Get feedback
    - [x] Make a spreadsheet of the actionable recommendations (add to wiki)
    - [x] Work with Product and Development to get issues created for those items
- [x] Review all documentation added to wiki and define what is missing

### Resources/Instructions
- Link to fill out the form and submit a win : [FORM](https://docs.google.com/forms/d/e/1FAIpQLSdF0MgS-M_Soi77qTp-MNfxMGqq85yqsHwvfHDngHAMyDQ8uQ/viewform)
- Link to the review sheet: [SHEET](https://docs.google.com/spreadsheets/d/1fXmYrmNtrgdzkM_odGIRbSwdaea3yS0cTLCJUzZ6Sc0/edit)
- Wiki page for researchers https://github.com/hackforla/website/wiki/UI-UX-Researcher

**Comment by Sihemgourou** [comment]
We will review this issue with @ExperimentsInHonesty before starting to work on this issue

**Comment by cwvivianlin** [comment]
Availability for this week: 3-4 hours this week
My estimated ETA for completing this issue: 2 weeks, i.e. March 23

**Comment by cwvivianlin** [comment]
@ExperimentsInHonesty @sacamp Would you be able to provide access to the Sheet in resources? Thank you!

Vivian

"Link to the review sheet: [SHEET](https://docs.google.com/spreadsheets/d/1fXmYrmNtrgdzkM_odGIRbSwdaea3yS0cTLCJUzZ6Sc0/edit)"

**Comment by ExperimentsInHonesty** [comment]
@cwvivianlin can you tell us what info you need from this sheet?  It has Personally Identifiable Information on it (PII), so we restrict access to it.  I have moved this issue to the in progress column.  Once you have answered the question, please move it to the Questions column.

**Comment by cwvivianlin** [comment]
> @cwvivianlin can you tell us what info you need from this sheet? It has Personally Identifiable Information on it (PII), so we restrict access to it. I have moved this issue to the in progress column. Once you have answered the question, please move it to the Questions column.

I understand--In this case, that's ok, I don't need to see the sheet to do the research. Thanks!

**Comment by cwvivianlin** [comment]
**Progress:**
-[research plan](https://docs.google.com/document/d/10pir-ln3-5KRkF4Ue3hpAh8nPYah9j_FMM7GHAK6zfE/edit) created. Would appreciate reviewing it with UXR/product.
-[Wiki page ](https://github.com/hackforla/website/wiki/Wins-submission-process-research)created.

**Blockers:** 
-Research plan review
-In the plan I listed some questions/concerns I have (highlighted in yellow)

**Next steps:** 
-Review research plan with UXR/product

**Availability:** 
-4 hours, Tue, Wed, Thurs.

**Comment by sacamp** [comment]
@cwvivianlin please use the following [template](https://docs.google.com/document/d/1EA5ZYHExu_RC8mfUAgxLxwJ-m_cMsGx_yAjE77gMXqw/edit) for your research plan. Create a copy and edit as needed. Thanks!

**Comment by cwvivianlin** [comment]
@sacamp  Thank you, will do! 

**Progress:**

Research plan updated: https://docs.google.com/document/d/10pir-ln3-5KRkF4Ue3hpAh8nPYah9j_FMM7GHAK6zfE/edit

I also created the script: https://docs.google.com/spreadsheets/d/1F4iiSmD0wFRZ4YHj8Y4-nooUZFmn0jlZriyG_eI8ZEs/edit#gid=0

**Comment by cwvivianlin** [comment]
@sacamp Would you like me to use Hfla calendly to recruit? Or I could also use my own. lmk. I'm ready to recruit. Thank you!

**Comment by cwvivianlin** [comment]
I am having difficulty recruiting the type of user I want. I sent out the Calendly link on Slack yesterday, so far no one has signed up.

I would like to recruit someone who is interested in sharing what've they've learned as a volunteer (the idea is that if they are interested in sharing, they are the people who might submit a Win). Doesn't matter if they are current or past volunteers. 5 total would be ideal. 

Options for next steps are:
-Reach out to individuals directly on Slack or Linkedin- Bonnie do you know anyone who might fit my criteria?
-Ask Bonnie to resend the slack recruiting message and tag everyone
-Expand criteria to any volunteer 

Thoughts? @sacamp @ExperimentsInHonesty

**Comment by sacamp** [comment]
@cwvivianlin please add the recruitment messaging to a comment in this issue.
Then, @ExperimentsInHonesty can use the text to message the general slack channel.
Additionally, we can reach out directly to individuals. Lets discuss this [spreadsheet](spreadsheet) at our next meeting.

**Comment by cwvivianlin** [comment]
@ExperimentsInHonesty This is the recruiting message, thank you for posting on the general slack channel:

Hi! I am a UX researcher on the HfLA website team. I am conducting user research on the “[Wins](https://www.hackforla.org/wins/)” page:trophy:, which features volunteer achievements in their own words (it could be gaining cross-function collaboration experience, etc. Wins don't have to be getting a full-time job). I'm looking for someone who is interested in sharing what you’ve learned as a volunteer so far (past and current volunteers welcome). I’d have you complete a few tasks and ask you questions. If you are interested, here’s the [LINK ](https://calendly.com/siteresearch/hack-for-la-wins-page-research?month=2022-03)to book a time over Zoom. Thank you for helping us improve the HfLA website!!:pray:

**Comment by cwvivianlin** [comment]
@ExperimentsInHonesty I am refining my script for my first user testing on Monday! I have additional questions. Can you tell me more about what you are trying to get to re: "how people would like to be notified when their win is accepted"? Besides email, is there any other means of notification you had in mind (Slack, etc.)? My thought is to show them a screenshot of a simple email telling them their Win has been published and asking their impression.

**Comment by cwvivianlin** [comment]
Progress: I've recruited 3 users! Need 2 more. First user testing starts tomorrow.

**Comment by ExperimentsInHonesty** [comment]
Yeah!!! ^

**Comment by cwvivianlin** [comment]
Progress: Testing with 7 users complete! I'm in the analysis phase. I will be able to present next Sunday, the soonest.

**Comment by cwvivianlin** [comment]
@sacamp Can you point me to the presentation template please? Thank you.

**Comment by cwvivianlin** [comment]
Progress: working on presentation slides. 
Find the presentation in the Wiki: https://github.com/hackforla/website/wiki/Wins-submission-process-research

**Comment by cwvivianlin** [comment]
Progress: Presentation slides and recommendation list complete. I plan to present in the 5/1 meeting (Sunday).

**Comment by cwvivianlin** [comment]
-Progress: Presented at Sunday meeting on 5/8.

-Need to put put recommendations into the "Action Steps" template: https://docs.google.com/spreadsheets/d/1NjrbicVDyd8aTXfPXUAdlX2xg9F3Ez9VThj4EwG-ouw/edit#gid=0 

-Bonnie suggested interviewing folks from Content, Data Science, Accessibility: We need to connect with the following people for your research [@Qiqi Zheng] [@Kiki](Accessibility) [@Karina Lopez][@Sophia Alice](Data Science) [@Zoe Zhong] [@Sam] (Content)

**Comment by cwvivianlin** [comment]
Progress:
-Action steps spreadsheet created
-Reviewed above spreadsheet with Bonnie

Todo:
-Based on comments from Bonnie, create issues to implement action steps

Blocker:
-I need access to this doc to learn how to write issues:
https://docs.google.com/document/u/0/d/1wYWr7uJ7dZeBwAAgksP-50_GXtCBQIywiqq5F5wQA6U/edit

**Comment by cwvivianlin** [comment]
Progress:
-Action steps issues created 
- #3139 
- #3143 
- #3140 
- #3142 
- #3145 

Ticket complete!
